dbms/src/Common/ProfileEvents.cpp

```diff
@@ -20,9 +20,9 @@
     M(ReadBufferAIOReadBytes, "") \
     M(WriteBufferAIOWrite, "") \
     M(WriteBufferAIOWriteBytes, "") \
-    M(ReadCompressedBytes, "") \
-    M(CompressedReadBufferBlocks, "") \
-    M(CompressedReadBufferBytes, "") \
+    M(ReadCompressedBytes, "Number of bytes read from compressed files.") \
```

Comment:
Not necessarily files. Also from network.

Reply:
Is it better now?

Reply:
Ok.

But the "number of uncompressed bytes read from ..." is not clear enough. The user may be confused and think that some bytes in compressed file are uncompressed and we count them.

Reply:
I've added clarifications.